front/app/containers/ProjectsShowPage/shared/header/ProjectActionButtons.tsx

```diff
@@ -131,54 +132,42 @@ const ProjectActionButtons = memo<Props>(({ projectId, className }) => {
 
   const { process_type, publication_status } = project.data.attributes;
 
-  const isProjectArchived =
-    project.data.attributes.publication_status === 'archived';
```

Comment:
publication_status is already extracted on line 132, so no need to repeat `project.data.attributes.`.